### PR TITLE
Update docs for developing on a remote host

### PR DESCRIPTION
**Pull request by vpchung**
In an effort to move away from the soon-to-be-deprecated shared AWS Sandbox account, I went through the steps of deploying an EC2 with the Service Catalog instead.  Success!:

Screen Shot 2022-06-10 at 12 53 57 AM

I made note of the steps I took and have updated the doc page accordingly.

### CODE REVIEW

File: docs/develop-on-a-remote-host.md
Comment (rrchai) [inline docs/develop-on-a-remote-host.md:172]:
```suggestion

- Follow the [**Storing GitHub credentials on the EC2 instance**](https://sagebionetworks.jira.com/wiki/spaces/APGD/pages/2590244872/Service+Catalog+Instance+Setup#Storing-GitHub-credentials-on-the-EC2-instance) instructions to config your GitHub account
```

@vpchung I was not able to push changes. I need to follow the ^ instruction to make it work - adding EC2 instance SSH key to my Github account. If you don't end up needing this step, please ignore this suggestion.

**Comment by tschaffter** [comment]
@vpchung You can merge this PR to `main` once Rong gave his approval. Thanks!

File: docs/develop-on-a-remote-host.md
Comment (rrchai) [inline docs/develop-on-a-remote-host.md:169]: `sudo usermod -aG docker ${USER}` I ran it during the workshop, so I am not sure if this is still needed. @vpchung 
 Did you need to run this CLI?

**Review by rrchai — APPROVED** [review]
Only two comments. Otherwise, look good to me!

File: docs/develop-on-a-remote-host.md
Comment (vpchung) [reply docs/develop-on-a-remote-host.md:169]: I never had to run this when I was setting up, since the instance image already configured Docker appropriately, so I'm not sure if this is needed?

I thought the error you encountered earlier was from the fork being out of sync?

File: docs/develop-on-a-remote-host.md
Comment (vpchung) [reply docs/develop-on-a-remote-host.md:172]: Good idea!

**Pull request merged**
